vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -466,7 +467,7 @@ export default class SbcHeader extends Mixins(NavigationMixin) {
 
   private checkAccountStatus () {
     // redirect if accoutn status is suspended
-    if (this.currentAccount?.accountStatus && this.currentAccount?.accountStatus === 'NSF_SUSPENDED') {
+    if ([AccountStatus.NSF_SUSPENDED, AccountStatus.SUSPENDED].some(status => status === this.currentAccount?.accountStatus)) {
       this.redirectToPath(this.inAuth, `${Pages.ACCOUNT_FREEZ}`)
     } else if (this.currentAccount?.accountStatus === 'PENDING_AFFIDAVIT_REVIEW') {
```

Comment:
Should use AccountStatus.PENDING_AFFIDAVIT_REVIEW here. Same to all other status code.